Makefile

```diff
@@ -59,7 +63,6 @@ all : runbuildprep lint image chart rptr_rpm test_rpm
 local: cms_meta_tools runbuildprep image chart_setup chart_package
 chart: chart_setup chart_package chart_test
 rptr_rpm: rptr_rpm_package_source rptr_rpm_build_source rptr_rpm_build
-test_rpm: test_rpm_package_source test_rpm_build_source test_rpm_build
```

Comment:
You should also remove the reference to test_rpm in the "all:" stanza a few lines above